napari/_vispy/layers/image.py

```diff
@@ -93,6 +93,18 @@ def __init__(
             layer_node_class=layer_node_class,
         )
 
+        self._array_like = True
+
+        # set the 3D shader step size bounds
+        self.min_step_size = 0.05
+        self.max_step_size = 20
+
+        # set the initial guess for a good step size
+        # while the camera is moving
+        self.initial_moving_step_size = 5
```

Comment:
Is this used anywhere? I never see it in the example reporting step_size and I don't see it in the code anywhere else?

Reply:
nope! nuking! 😂

Reply:
yes I think that's what this was originally for. But I need to think about the best way to use this...